tests/gtest_dlt_daemon_gateway.cpp

```diff
@@ -456,8 +456,8 @@ TEST(t_dlt_gateway_parse_get_log_info, normal)
     char ecuid[] = "ECU2";
     uint32_t sid = DLT_SERVICE_ID_GET_LOG_INFO;
     uint8_t status = 7;
-    uint16_t count_app_ids = 1;
-    uint16_t count_context_ids = 1;
+    uint16_t count_app_ids = DLT_HTOLE_16(1);
```

Comment:
Hello @LocutusOfBorg 
FYI
I remove the preprocessor for BYTE_ORDER here.

Reply:
thanks